core/Symbols.cc

```diff
@@ -260,6 +260,11 @@ SymbolRef Symbol::findMemberTransitiveInternal(const GlobalState &gs, NameRef na
 vector<Symbol::FuzzySearchResult> Symbol::findMemberFuzzyMatch(const GlobalState &gs, NameRef name,
                                                                int betterThan) const {
     vector<Symbol::FuzzySearchResult> res;
+    // Don't run under the fuzzer, as otherwise fuzzy match dominates runtime.
+    if (gs.runningUnderFuzzer) {
```

Comment:
While we did historically find bugs here with fuzzer, I'm +1 for disabling it now.

Reply:
Can we leave a comment here indicating that this _could_ run under fuzzer and one day it could be benefitical